src/app/active-mode/Views/current-activity/VendorBounties.tsx

```diff
@@ -51,26 +77,20 @@ function VendorBounties({
   ownedItemHashes,
   account,
 }: Props) {
+  const [bountyFilters, setBountyFilters] = useState<BountyFilter[]>([]);
```

Comment:
Use the `PursuitsGroup` component instead, it manages all this state for you.

Reply:
perfect, updated to use that.